readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -933,6 +933,54 @@ def test_webhook_doesnt_build_latest_if_is_deactivated(self, trigger_build):
         self.assertEqual(resp.status_code, 200)
         self.assertFalse(resp.data['build_triggered'])
 
+    def test_webhook_build_latest_and_master(self, trigger_build):
+        client = APIClient()
+        integration = Integration.objects.create(
+            project=self.project,
+            integration_type=Integration.API_WEBHOOK,
+        )
+
+        latest_version = self.project.versions.get(slug=LATEST)
+        default_branch = self.project.versions.get(slug='master')
+
+        self.assertTrue(latest_version.active)
+        self.assertTrue(default_branch.active)
```

Comment:
Can you add another test (if it's not already) that test when `latest` is inactive but not the `default_branch` that the `latest` version is not triggered for build an is not re-activated and the `default_branch` is triggered?